eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
So did a grep for use __DIR__ in other tests, and it's all over the place for `inputUri`, so I say we keep this as is for now and consider looking into if this is wrong or not later.